common/presenters/events-to-timeline/event-details-helpers.js

```diff
@@ -0,0 +1,84 @@
+const pluralize = require('pluralize')
+
+const { event: eventModel } = require('../../../common/lib/api-client/models')
+const i18n = require('../../../config/i18n')
+
+const eventRelationships = Object.keys(eventModel.fields).filter(field => {
+  return typeof eventModel.fields[field] === 'object'
+})
+
+// add resources that belong to move and are common to all events
+// NB. move.supplier will exist even when event.supplier does not
+const getMoveDetails = move => ({
+  move,
+  person: move.profile?.person,
+})
+
+// if an event has a supplier, the supplier was the agent
+// if not, check the locales for a default agent
+// finally, use PMU as default agent
+const getEventAgency = event => {
+  let agency = event.supplier ? 'supplier' : undefined
+
+  if (!agency) {
+    const defaultAgencyKey = `events::${event.event_type}.defaultAgency`
+    agency = i18n.exists(defaultAgencyKey) ? i18n.t(defaultAgencyKey) : 'pmu'
```

Comment:
Could we have the `pmu` string defined in config somewhere? Magic strings in codebases have a tendency to go wild